documentation/correctness/issues.adoc

```diff
@@ -0,0 +1,96 @@
+= Issues
```

Comment:
I _think_ during the discussion of https://github.com/LionWeb-io/lionweb-mps/pull/89 we gravitated towards the term `finding`. Should we use the same here?